graphql/contactmethod.go

```diff
@@ -170,77 +170,3 @@ func (h *Handler) sendContactMethodTest() *g.Field {
 		},
 	}
 }
-
-func (h *Handler) sendContactMethodVerification() *g.Field {
```

Comment:
deleting these as the return types have changed, and isn't worth refactoring to bring it to a usable state.